arm-servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```

Comment:
I have few ones that the model name isn't exactly the same as the wire name. Do I just need "x-ms-discriminator-value" for those ones, or do I need for all in this case?